### PR TITLE
Test helpers: has_sybase(), on_head_network() and is_32_bit() + version 4.3

### PR DESCRIPTION
**Pull request by taldcroft**
### MacOS laptop:
```
$ ipython
>>> from testr.test_helper import *
>>> has_sybase()
False
>>> on_head_network()
False
```
### kadi and fido
```
$ ipython
>>> from testr.test_helper import *
>>> has_sybase()
True
>>> on_head_network()
True
```
### c3po-v  56 subnet.  is this "on HEAD"?
```
$ ipython
>>> from testr.test_helper import *
>>> has_sybase()
True
>>> on_head_network()
False
```

### MacOS laptop
```
In [1]: from testr.test_helper import *

In [2]: is_32_bit()
Out[2]: False
```

See https://github.com/sot/eng_archive/pull/175

### CODE REVIEW

**Comment by jeanconn** [comment]
c3po-v is "cross-mounted?"  IIRC .
So maybe it is an OPS LAN machine that can see /proj/sot/ska.

File: testr/test_helper.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    Return True if the system apparently can run Sybase queries from Python.
+
+    In detail, this is True if the SYBASE and SYBASE_OCS env variables are set
+    and the correct Python shared object library exists on the system.
+    """
+    try:
+        path = Path(os.environ['SYBASE'],
+                    os.environ['SYBASE_OCS'],
+                    'python', 'python34_64r', 'lib', 'sybpydb.so')
+    except KeyError:
```
Comment (jeanconn) [inline testr/test_helper.py]: I think on GRETA the keys will be defined in production environments but the dir won't exist.

File: testr/test_helper.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        # If either env var not defined then there is no Sybase
+        out = False
+    else:
+        out = path.exists()
```
Comment (jeanconn) [inline testr/test_helper.py]: Which is consistent with this usage.

**Comment by taldcroft** [comment]
So this doesn't get stalled and lost, can you re-review @jeanconn ?

**Comment by jeanconn** [comment]
Besides the c3po-v weirdness (is on the head network what we care about checking or a more comprehensive review of available resources?), this looks good to me.  I also checked it on chimchim and a 32 bit GRETA machine and see the expected outputs.

File: testr/runner.py [hunk 3]
```diff
@@ -12,7 +12,7 @@ class TestError(Exception):
 
 
 def testr(*args, **kwargs):
-    """
+    r"""
```
Comment (jeanconn) [inline testr/runner.py]: I forgot to ask, what's the deal with the "r"s?

Comment (jeanconn) [reply testr/runner.py]: Eh. I suppose you had some display reason for changing these to raw strings.  Didn't really line up with rest of the intent of the PR but not an issues for me and already merged.

**Pull request merged**